### ISSUE TITLE
Improve Swiftype result excerpts

### ISSUE THREAD

**Issue by sawyerh** [siteleaf/learn.siteleaf.com#42]
These excerpts are all :-1: 

![image](https://cloud.githubusercontent.com/assets/371943/13893792/b15edf18-ed37-11e5-88cd-38f332ad8daf.png)

**Comment by sskylar** [comment]
Add `data-swiftype-index="false"` to any elements you want it to ignore...

```html
  <div id="nav" data-swiftype-index="false">
    This (and all child) content will be excluded.
  </div>
```

https://swiftype.com/documentation/content_inclusion_exclusion

**Comment by eturk** [comment]
It seems like the sidebar is what's causing the extra results. Fixed with 792d73babd3ed6fc621aabb1f02dd691c3ee8bd7.
